sns_ik_lib/CMakeLists.txt

```diff
@@ -39,7 +39,8 @@ add_library(sns_ik
             src/fsns_velocity_ik.cpp
             src/osns_sm_velocity_ik.cpp
             src/osns_velocity_ik.cpp
-            src/sns_ik_data_utilities.cpp
+            src/rng_utilities.cpp
+            src/robot_model.cpp
```

Comment:
Same issue as above - I think that I'm missing something fundamental about how ROS package API works. Let's talk off-line so that I first understand the situation better.

Reply:
fixed in the new commit.